scripts/easy_train.py

```diff
@@ -1035,7 +1055,8 @@ def setup_c_chess_cli(directory):
     with open(os.path.join(directory, 'make.py', '-c', 'gcc'), 'w') as makefile:
```

Comment:
This seems wrong, `-c` and `gcc` should be command options, not folders.

Reply:
pretty dumb, fixed